crates/prover/benches/prover_speed_benchmark.rs

```diff
@@ -68,5 +69,99 @@ fn fibonacci_prove_benchmark(c: &mut Criterion) {
     group.finish();
 }
 
-criterion_group!(benches, fibonacci_prove_benchmark);
+// =============================
+// SHA-256 (1024-byte message)
+// =============================
+
+/// Compile the SHA-256 Cairo-M example program
+fn compile_sha256() -> Program {
```

Comment:
prepare_sha256_input_1kb and compile_sha256 are used by both benchmarks. Factorize to avoid maintain both?